filesystem/uimage.ksy

```diff
@@ -0,0 +1,103 @@
+meta:
+  id: u_image
+  endian: be
```

Comment:
`meta/title` and top-level `doc` element would be very welcome here to raise exposure of this spec and make it possible for other people to find it using search engines, etc.